frameworks/helloworld/tests/scale/test_scale.py

```diff
@@ -0,0 +1,72 @@
+import pytest
+import logging
+import sdk_install
+import sdk_security
+import sdk_dcos
+from tests import config
+
+log = logging.getLogger(__name__)
+
+ACCOUNTS = {}
+
+
+@pytest.mark.scale
+def test_scaling_load(service_count,
+                      scenario) -> None:
+    """Launch a load test scenario. This does not verify the results
+    of the test, but does ensure the instances and jobs were created.
+
+    Args:
+        count: number of helloworld services to install
+        scenario: yaml scenario to run helloworld with (normal, crashloop) are added for this case
+    """
+    security_mode = sdk_dcos.get_security_mode()
+    # TODO: parallelize account creation and installation if time is an issue in scale tests
+    for index in range(service_count):
+        service_name = "{}-{}-{}".format(config.PACKAGE_NAME, scenario, index)
+        _create_service_account(service_name,
+                                security_mode)
+        _install_service(service_name,
+                         scenario,
+                         security_mode)
+
+
+def _install_service(service_name, scenario, security=None):
```

Comment:
With the other changes:
```suggestion
def _install_service(service_name, scenario, security_info):
    # do not wait for deploy plan to complete, all tasks to launch or marathon app deployment
    # supports rapid deployments in scale test scenario
    options = {"service": {"name": service_name, "yaml": scenario}}
    if security_info:
        options["service"]["service_account"] = security_info["name"]
        options["service"]["service_account_secret"] = security_info["secret"]

    sdk_install.install(
        config.PACKAGE_NAME,
        service_name,
        config.DEFAULT_TASK_COUNT,
        additional_options=options,
        wait_for_deployment=False,
        wait_for_all_conditions=False
    )
```

Reply:
Looking at this, it may even make sense to change the keys for the `security_info` dict in `sdk_security` to `service_account` and `service_account_secret` so that the key mapping isn't even required.